doc/build.txt

```diff
@@ -18,6 +19,9 @@ all/the specified platforms.
     --emulator ......................... Deploy to an emulator
     --target ........................... Deploy to a specific target
 
+    --buildConfig....................... Use the specified build configuration
```

Comment:
Addressed both comments - added an example and fixed the grammar definition on line 9.